src/tests/__snapshots__/app.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { MemoryRouter as Router } from 'react-router-dom';
+import store from '../../redux/store';
+import App from '../../App';
+
+describe('Testing App component', () => {
+  test('Snapshot testing', () => {
+    const myRenderer = renderer.create(
+      <Provider store={store}>
+        <Router>
+          <App />
+        </Router>
+        ,
+      </Provider>,
+    );
+    const testComponent = myRenderer.toJSON();
+    expect(testComponent).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Testing: In reading the requirement of your Project, I noticed that your testing did not cover 100% of the requirements. Therefore, I would advise you to:
- to write the unit tests for several components of the app and some pure functions in Redux code.
- to write integration tests using React Testing Library. 

- [ ] I know you can make it, keep trying 💪 

![Capture](https://user-images.githubusercontent.com/63972670/225521760-7316b662-3633-4d3e-a2cc-d892bd002252.PNG)